components/brave_sync/client/bookmark_change_processor.cc

```diff
@@ -371,7 +371,14 @@ void BookmarkChangeProcessor::BookmarkNodeChanged(BookmarkModel* model,
 
 void BookmarkChangeProcessor::BookmarkMetaInfoChanged(
     BookmarkModel* model, const BookmarkNode* node) {
-  BookmarkNodeChanged(model, node);
```

Comment:
Seem like we no longer need `ScopedPauseObserver`
and also since `BookmarkMetaInfoChanged` is ignored, we also need to call `BookmarkNodeChanged` in `BookmarkNodeMoved`

Reply:
Yes, indeed, we had used `ScopedPauseObserver` just to avoid processing of our initiated events when we set a metadata .

Reply:
But still need `ScopedPauseObserver` in `BookmarkChangeProcessor::ApplyChangesFromSyncModel` .

Reply:
Updated PR @darkdh